### PR TITLE
Update libsndfile

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -108,8 +108,8 @@ The repository consists mostly of externally hosted subrepositories:
 .. _libtiff: https://gitlab.com/libtiff/libtiff
 .. |libtifflic| replace:: BSD-2 license
 .. _libtifflic: https://gitlab.com/libtiff/libtiff/-/blob/master/README.md
-.. |libtiffver| replace:: 4.4.0 (+ Build System Patch)
-.. _libtiffver: https://gitlab.com/libtiff/libtiff/-/tree/v4.4.0
+.. |libtiffver| replace:: 4.5.0 (+ Build System Patch)
```
Comment (JanuszL) [inline README.rst:111]: I think it has been done already in https://github.com/NVIDIA/DALI_deps/pull/74/files ?

File: README.rst [hunk 2]
```diff
@@ -54,12 +54,12 @@ The repository consists mostly of externally hosted subrepositories:
...
 .. _libsndfile: https://github.com/libsndfile/libsndfile
-.. |libsndfilever| replace:: 1.1.0
-.. _libsndfilever: https://github.com/libsndfile/libsndfile/releases/tag/1.1.0
+.. |libsndfilever| replace:: 1.2.0
+.. _libsndfilever: https://github.com/libsndfile/libsndfile/releases/tag/1.2.0
 .. |libsndfilelic| replace:: LGPL v2.1 license
 .. _libsndfilelic: https://github.com/libsndfile/libsndfile/blob/master/COPYING
 .. |libsndfilesrc| replace:: (Source Snapshot)
-.. _libsndfilesrc: https://developer.download.nvidia.com/compute/redist/nvidia-dali/libsndfile-1.1.0.tar.gz
+.. _libsndfilesrc: https://developer.download.nvidia.com/compute/redist/nvidia-dali/libsndfile-1.2.0.tar.gz
```
Comment (JanuszL) [inline README.rst:62]: Please upload.

Comment (jantonguirao) [reply README.rst:62]: Done

File: README.rst (hunk 1 above)
Comment (jantonguirao) [reply README.rst:111]: True, don't know what happened. Will update

**Review by JanuszL — APPROVED** [review]

**Review by awolant — APPROVED** [review]

**Pull request merged**